sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/Listeners/ServiceBusListenerTests.cs

```diff
@@ -206,6 +207,13 @@ public void GetMonitor_ReturnsExpectedValue()
             Assert.AreSame(scaleMonitor, scaleMonitor2);
         }
 
+        [Test]
+        public void StopAsync_LogListenerDetails()
+        {
+            Assert.ThrowsAsync<InvalidOperationException>(() => _listener.StopAsync(CancellationToken.None));
+            Assert.NotNull(_loggerProvider.GetAllLogMessages().SingleOrDefault(x => x.FormattedMessage.StartsWith("ServiceBus listener stopped")));
```

Comment:
What are we actually testing here? This would have passed prior to the changes here I believe.

Reply:
we are testing that a new log exists on ServiceBus listener stop.